src/ContainerBuilder.php

```diff
@@ -234,9 +248,16 @@ protected function findConfigFile($file)
         throw new LoaderException(sprintf("The config file '%s' does not exist in any of the configured paths", $file));
     }
 
-    protected function findImportedConfigFile($file, $dir)
+    /**
+     * @param $file
+     * @param $dir
+     * @param $relativeDir
+     * @return string
+     * @throws LoaderException
+     */
+    protected function findImportedConfigFile($file, $dir, $relativeDir)
     {
-        $filePath = $dir . "/" . $file;
+        $filePath = $dir . DIRECTORY_SEPARATOR . $relativeDir . DIRECTORY_SEPARATOR . $file;
```

Comment:
This works (obvs) but we need to maintain BC if possible, otherwise some library config may stop working.

I think our best option is to check the relative path, then the old non-relative path as well

Reply:
We'd have to look at the exception message too

Reply:
Have a look if `$relativeDir` could be an optional arg

Reply:
Good point, I'd forgotten about that case. I'll add a double check